### PR TITLE
Add Hishigata (clone) and Project Rhombus (original)

### PR DESCRIPTION
**Pull request by crse**

### CODE REVIEW

File: games/h.yaml [hunk 1]
```diff
@@ -344,6 +344,23 @@
...
+  - Project Rhombus
+  type: clone
+  repo: 'https://github.com/LumpBloom7/hishigata'
+  development: sporadic
+  status: playable
+  content: open
+  langs:
+  - 'C#'
+  frameworks:
+  - osu!framework
```
Comment (cxong) [inline games/h.yaml:358]:
```suggestion
  - osu!
```

Comment (cxong) [reply games/h.yaml:358]: also add this value to frameworks in dangerfile.js to silence the warning

**Review by cxong — CHANGES_REQUESTED** [review]

File: games/h.yaml (hunk 1 above)
Comment (crse) [reply games/h.yaml:358]: I believe we should match the official name instead: `osu!framework` -> https://github.com/ppy/osu-framework

**Comment by crse** [comment]
Should be good right now :)

**Pull request merged**